src/Stratis.Bitcoin.Features.Wallet/Validations/IsBitcoinAddressAttribute.cs

```diff
@@ -7,6 +7,11 @@ public class IsBitcoinAddressAttribute : ValidationAttribute
     {
         protected override ValidationResult IsValid(object value, ValidationContext validationContext)
         {
+            if (!this.Required)
```

Comment:
I see that `MoneyFormatAttribute`, for instance, uses a parameter in its constructor instead. But this is fine